src/main/scala/io/iohk/ethereum/db/dataSource/RocksDbDataSource.scala

```diff
@@ -37,16 +35,18 @@ class RocksDbDataSource(
     * @return the value associated with the passed key.
     */
   override def get(namespace: Namespace, key: Key): Option[Value] = {
-    assureNotClosed()
-    RocksDbDataSource.dbLock.readLock().lock()
+    dbLock.readLock().lock()
```

Comment:
Is the change from global lock: `RocksDbDataSource.dbLock.readLock().lock()`
to local specific to class instance: `dbLock.readLock().lock()`
Intended. Here and below?

Reply:
From what I see it's still the global one used, it's just imported from companion object now

Reply:
Yes, right